docs/maintainer-guide/issues.md

```diff
@@ -20,39 +20,60 @@ There are four primary issue categories:
+        * For all issues, please add labels describing the part of ESLint affected:
+            * "3rd party plugin" - related to third-party functionality (plugins, parsers, rules, etc.)
             * "build" - related to commands run during a build (testing, linting, release scripts, etc.)
             * "cli" - related to command line input or output, or to `CLIEngine`
             * "core" - related to internal APIs
             * "documentation" - related to content on eslint.org
             * "infrastructure" - related to resources needed for builds or deployment (VMs, CI tools, bots, etc.)
-1. Once it's clear what type of issue it is, make sure all of the relevant information is provided:
-    * **Bugs**: See [bug reporting guidelines](/docs/developer-guide/contributing/reporting-bugs.md)
-    * **New Rules:** See [rule proposal guidelines](/docs/developer-guide/contributing/new-rules.md)
-    * **Rule Changes:** See [rule change proposal guidelines](/docs/developer-guide/contributing/rule-changes.md)
-    * **Other Changes:** See [change proposal guidelines](/docs/developer-guide/contributing/changes.md)
+            * "rule" - related to core rules
+        * Move the issue to the "Ready for Dev Team" column in the Triage project
+
+## Dev Team Process
+
+When an issue has been moved to the "Ready for Dev Team" column, any dev team member can pick up the issue to start evaluating it.
+
+1. Move the issue into the "Evaluating" column.
 1. Next steps:
-    * **Questions:** answer the question and close the issue when the conversation is over.
     * **Bugs:** if you can verify the bug, add the "accepted" label and ask if they would like to submit a pull request.
     * **New Rules:** if you are willing to champion the rule (meaning you believe it should be included in ESLint core and you will take ownership of the process for including it), add a comment saying you will champion the issue, assign the issue to yourself, and follow the [guidelines](#championing-issues) below.
     * **Rule Changes:** if you are willing to champion the change and it would not be a breaking change (requiring a major version increment), add a comment saying that you will champion the issue, assign the issue to yourself, and follow the [guidelines](#championing-issues) below.
     * **Breaking Changes:** if you suspect or can verify that a change would be breaking, label it as "Breaking".
     * **Duplicates:** if you can verify the issue is a duplicate, add a comment mentioning the duplicate issue (such as, "Duplicate of #1234") and close the issue.
 1. Regardless of the above, always leave a comment. Don't just add labels, engage with the person who opened the issue by asking a question (request more information if necessary) or stating your opinion of the issue. If it's a verified bug, ask if the user would like to submit a pull request.
+1. If an RFC is required, move the issue to the "RFC Needed" column and comment on the issue that an RFC is needed.
```

Comment:
Should we add the "needs rfc" label instead, as mentioned in https://github.com/eslint/eslint-github-bot/issues/136 ?

Reply:
Not instead, no. The project is meant to show where the issue is in the acceptance process, so having a column indicates that we are waiting for an RFC. (We could rename it “Waiting for RFC” if that helps.)

I’m not sure we need a label anymore.

Reply:
Make sense to me, we should just update tasks in https://github.com/eslint/eslint-github-bot/issues/136#issue-786478245 if we won't have the label.

Reply:
I'd rather wait a bit before saying we're going to remove the label. As I said, I'm not sure if we need it, but maybe we do? In the original task list it appears I thought we'd still use it. I guess the question is if we want a way to flag that a particular issue needs an RFC vs. saying that the next step in process is to create an RFC?